src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -66,6 +72,32 @@ public GridStorageCache(final IGrid g) {
     public void onUpdateTick() {
         this.itemMonitor.onTick();
         this.fluidMonitor.onTick();
+
+        this.itemBytesTotal = 0;
+        this.itemBytesUsed = 0;
+        try {
+            for (ICellProvider icp : this.activeCellProviders) {
+                if (icp.getClass() == Class.forName("appeng.tile.storage.TileDrive")) {
```

Comment:
Too many direct castings that can fail some how. It is better to make a check with instanceof to be sure that all going fine